.cirrus.yml

```diff
@@ -95,13 +95,17 @@ task:
         - name: sid-newkernel
         - name: sid-newkernel-uring
 
-    - env:
+    - name: windows-ci-vs-2019-VM
+      env:
         PACKERFILE: packer/windows.pkr.hcl
         SCRIPTS: scripts/windows*
+        TASK_NAME: windows-ci-vs-2019
 
-      matrix:
-        - name: windows-ci-vs-2019
-        - name: windows-ci-mingw64
+    - name: windows-ci-mingw64-VM
+      env:
+        PACKERFILE: packer/windows.pkr.hcl
+        SCRIPTS: scripts/windows*
+        TASK_NAME: windows-ci-mingw64
```

Comment:
I was imagining that we wouldn't set 'name' directly anymore, but instead have something like this:
```yaml
name: Building VM Image: $IMAGE_NAME
name: Building Docker Image: $IMAGE_NAME
```
in the respective tasks.

Reply:
Then depend_on should change. I created a draft PR to see changes: https://github.com/anarazel/pg-vm-images/pull/54